control-flow-exercises.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Created by daniel.potts on 20/08/15.
+ */
+//Create a triangle using astrix.
+var numLines = 2;
+var stars = ("*");
```

Comment:
First: String literals should be '*' not "*"
Second, don't declare like this. I know they drum "no magic strings" into you at uni - but in reality it's only important to avoid those if it isn't clear from the context what is happening. Here it is *without* the variable.